docs/doc/14-sql-commands/00-ddl/101-network-policy/ddl-alter-policy.md

```diff
@@ -0,0 +1,36 @@
+---
+title: ALTER NETWORK POLICY
+---
+
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.26"/>
+
+Modifies an existing network policy in Databend. 
+
+## Syntax
+
+```sql
+ALTER NETWORK POLICY policy_name
+    SET ALLOWED_IP_LIST=('allowed_ip1', 'allowed_ip2', ...)
+    BLOCKED_IP_LIST=('blocked_ip1', 'blocked_ip2', ...)
```

Comment:
`ALLOWED_IP_LIST` and `BLOCKED_IP_LIST` are also optional. Users can modify only one of the items while leaving the others unchanged.